content/en/contributing.md

```diff
@@ -0,0 +1,37 @@
+{
+    "title": "Contributing",
+    "weight": 50
+}
+
+First of all, thank you very much for your interest in contributing! We warmly welcome contributions and will do our best to help you. On this page, we want to give you an overview of how our project works and how you can contribute.
+
+We invite you to chat with us through Matrix, either through the specific [Tweasel room](https://matrix.to/#/#dade-tweasel:matrix.altpeter.me) or the [space](https://matrix.to/#/#datenanfragen:matrix.altpeter.me) for datarequests.org. Feel free to ask questions, pitch your ideas, or just talk with the community. Alternatively, you can also reach us via [email](mailto:dev@datenanfragen.de).
+
+## Project governance and mission
+
+Tweasel is run and maintained by [Datenanfragen.de e. V.](https://www.datarequests.org/verein/), a non-profit organsation from Germany on a [mission](https://www.datarequests.org/verein/mission-statement/) to advance data protection and privacy in Europe and beyond. In doing so, we stand firmly on the side of the consumer and citizen, whose rights we defend. We manage various projects, including our main site [datarequests.org](https://www.datarequests.org), where we help users exercise their GDPR rights.
```

Comment:
```suggestion
Tweasel is run and maintained by [Datenanfragen.de e. V.](https://www.datarequests.org/verein/), a non-profit organsation from Germany on a [mission](https://www.datarequests.org/verein/mission-statement/) to advance data protection and privacy in Europe and beyond. In doing so, we stand firmly on the side of the consumers and citizen, whose rights we defend. We manage various projects, including our main site [datarequests.org](https://www.datarequests.org), where we help users exercise their GDPR rights.
```

Reply:
This should probably also mention funding and who makes decisions in the end?

Reply:
The "consumer and citizen" is verbatim from the mission statement on dr.org. I think it does also work that way. Otherwise, you'd also have to say "consumers and citizen**s**", wouldn't you?

Reply:
> This should probably also mention funding and who makes decisions in the end?

I've been wanting to have formalised guidelines for such processes for a while. So far, we never did, as far as I'm aware. :P

Reply:
Yeah, you are right, I read too quickly.

Reply:
> I've been wanting to have formalised guidelines for such processes for a while. So far, we never did, as far as I'm aware. :P

Yes, but de facto there is some kind of hierarchy. We should mention that here, shouldn‘t we?

Reply:
> Yes, but de facto there is some kind of hierarchy. We should mention that here, shouldn‘t we?

Are you talking about the suggestion I accepted below or do you want to go into further details?

I'm fine with the suggestion but I wouldn't know what else to say, since I genuinely don't really know myself what our guidelines would be.

Reply:
No, that‘s exactly what I was talking about.